pkg/cli/cmd/workspace/create/create.go

```diff
@@ -177,7 +177,7 @@ func (r *Runner) Validate(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		_, err := client.ShowUCPGroup(cmd.Context(), "local", group)
+		_, err := client.GetResourceGroup(cmd.Context(), "local", group)
```

Comment:
Is it better to call this GetRadiusResourceGroup or add a comment due to confusing terminology with az resource group?

Reply:
I think it's much better to just say `ResouceGroup`. The context makes it clear that is about the Radius feature.